desktop-app/app/constants/browserSync.js

```diff
@@ -0,0 +1,5 @@
+export const BROWSER_SYNC_PORT = 12719;
+export const BROWSER_SYNC_VERSION = '2.26.7';
+
+export const BROWSER_SYNC_HOST = `localhost:${BROWSER_SYNC_PORT}`;
+export const BROWSER_SYNC_EMBED_SCRIPT = `https://${BROWSER_SYNC_HOST}/browser-sync/browser-sync-client.js?v=${BROWSER_SYNC_VERSION}`;
```

Comment:
@esprush Is the version mandatory here?

Reply:
@manojVivek works without the version also. was not sure of the exact implications of removing it, so kept exactly as it was suggested.

Reply:
@manojVivek works without the version also. was not sure of the exact implications of removing it, so kept exactly as it was suggested.